contracts/bulkers/MainnetBulker.sol

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.15;
+
+import "./BaseBulker.sol";
+import "./IWstETH.sol";
+
+contract MainnetBulker is BaseBulker {
+    address payable public immutable steth;
+    address payable public immutable wsteth;
+
+    bytes32 public constant ACTION_SUPPLY_STETH = "ACTION_SUPPLY_STETH";
+    bytes32 public constant ACTION_WITHDRAW_STETH = "ACTION_WITHDRAW_STETH";
+
+    constructor(
+        address admin_,
+        address payable weth_,
+        address payable steth_,
+        address payable wsteth_
+    ) BaseBulker(admin_, weth_) {
+        steth = steth_;
```

Comment:
You can derive this from `wsteth.stETH()`.

Reply:
good call; updated (and made the `steth` and `wsteth` address not `payable`)

Reply:
Good catch, didn't catch the `payable` on my end